src/Pages/AuthenticationPages/Login.jsx

```diff
@@ -1,22 +1,77 @@
-import React from "react"
+import React, { useState } from "react"
 import "./UserAuth.css"
-import { Link } from "react-router-dom"
+import { Link, useNavigate } from "react-router-dom"
+import axios from "axios"
+import { useToast } from "../../Context/toast-context"
+import { useUserLogin } from "../../Context/user-login-context"
 
 function Login()
 {
+    const { setUserLoggedIn } = useUserLogin()
+    const { showToast } = useToast()
+
+    const [userEmail    , setUserEmail]    = useState('')
+    const [userPassword , setUserPassword] = useState('')
```

Comment:
Can be changed into 1 useState and passing it as object `{userEmail:"",userPassword:""}`

Reply:
I am using them in same file and not passing anywhere else.
So creating object will lead to extra destructuring step.